day_2/project_of_the_day.py

```diff
@@ -0,0 +1,18 @@
+# https://replit.com/@appbrewery/tip-calculator-end
+
+GREETING = 'Welcome to the tip calculator!'
+ASK_TOTAL_BILL = 'What was the total bill?'
+ASK_TIP_PERCENT = 'How much tip would you like to give? 10, 12, or 15?'
+ASK_NUMBER_OF_PEOPLE = 'How many people to split the bill?'
+TELL_THE_RESULT = 'Each person should pay: ${:.2f}'
+
+print(GREETING)
+
+bill = float(input(ASK_TOTAL_BILL))
+tip_percent = int(input(ASK_TIP_PERCENT))
```

Comment:
try/except would be nice in case of invalid input

Reply:
I`ll fix it, thanks!

Reply:
:)